### PR TITLE
feat: allow to uninstall Helm releases

### PR DESCRIPTION
**Pull request by matiaszilli**
This adds the following features:
- Uninstall Helm releases.
- Remove namespace when uninstalling a Helm release.

### CODE REVIEW

**Comment by matiaszilli** [comment]
@Silvia-N

File: README.md [hunk 1]
```diff
@@ -21,12 +21,14 @@ Following inputs can be used as `step.with` keys
...
 | `chart-path`      | String  | The path to the chart. (defaults to `helm/`) |
+| `uninstall`      | String  | Whether to uninstall a Helm release. (defaults to `false`) |
+| `delete-namespace`      | String  | Whether to remove the namespace when uninstall a Helm release |
 
 
 ## Example usage
 
 ```yaml
-uses: craftech-io/eks-helm-deploy-action@v1
+uses: recurse-id/eks-helm-deploy-action@master
```
Comment (Silvia-N) [inline README.md:31]: should we move this to nirvana-health @recurse-id ?

File: deploy.sh [hunk 2]
```diff
@@ -2,19 +2,35 @@
...
     aws eks \
-        --region ${AWS_REGION} \
-        update-kubeconfig --name ${CLUSTER_NAME} 
+        --region "${AWS_REGION}" \
+        update-kubeconfig --name "${CLUSTER_NAME}" 
+fi
+
+# Helm Uninstall
+UNINSTALL_COMMAND="helm uninstall ${DEPLOY_NAME} --timeout ${TIMEOUT}"
+DELETE_NAMESPACE_COMMAND="kubectl delete ns ${DEPLOY_NAMESPACE}"
```
Comment (Silvia-N) [inline deploy.sh:22]: DEPLOY_NAME and DEPLOY_NAMESPACE do not have spaces, right?

File: README.md [hunk 3]
```diff
@@ -21,12 +21,14 @@ Following inputs can be used as `step.with` keys
 | `name`      | String  | The name of the helm release |
 | `flags`      | String  | Additional flags to pass to Helm |
 | `chart-path`      | String  | The path to the chart. (defaults to `helm/`) |
+| `uninstall`      | String  | Whether to uninstall a Helm release. (defaults to `false`) |
+| `delete-namespace`      | String  | Whether to remove the namespace when uninstall a Helm release |
```
Comment (Silvia-N) [inline README.md]:
```suggestion
| `delete-namespace`      | String  | Whether to remove the namespace when uninstalling a Helm release (defaults to `false`) |
```

**Review by Silvia-N — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (matiaszilli) [reply README.md:31]: It must to be a public repository so we decided to host it here for security reasons.

File: deploy.sh (hunk 2 above)
Comment (matiaszilli) [reply deploy.sh:22]: Yes, they don't have spaces

**Pull request merged**